src/activities/components/ActivityItem.vue

```diff
@@ -170,6 +170,7 @@ export default {
     },
     detail (event) {
       if (event.target.closest('a')) return // ignore actual links
+      if (event.target.classList.contains('user-slot-wrapper')) return // ignore actual links
```

Comment:
I'm trying to understand this bit, which means we could do with more comments (including the `closest('a')` bit, which even though I added, I forgot the reason... 

I recalled it though after thinking/reading a bit more - it's because there are some `<a>` links inside the activity item (e.g. link to the place), and we don't want to trigger the detail view (sidebar activity chat) when we click that link. So I guess the comment is kind of right, but still needed some thinking!

As for the second case though, I was confused, and admittedly I didn't read your main PR comment before reading the code! So I spent some time trying to understand what was going on.

You could use the explanation from your PR description as the comment:

> don't go to activity detail when clicking on an empty slot

But... why are the clicks on the `.user-slot-wrapper` element making it up to here? I thought the `.stop` modifier on the click handler in `UserSlot.vue` should prevent propagation of the event?

I wonder if returning `null` in the case `showJoin` is false prevents the `.stop` from working?

Reply:
Oh yeah, forgot to add that comment in that line^^ sorry.
About the propagation: I was confused myself that this still propagated. Good point at the `null` thing. Will try again.
About the why of this: for me, it seems not logical that clicking on a element that looks disabled does something (in this case, opening the chat window).

Reply:
Fixed propagation with latest commit. The empty slots have their own component (`EmptySlot.vue`) and there was no `@click.stop` so thats why clicks on them propagated... This is now fixed and I therefore removed the `event.target.classList.contains(...)` in `ActivityItem.vue`.